llpc/test/shaderdb/general/PipelineRays_TestLgcRtTraceRayOp.pipe

```diff
@@ -0,0 +1,110 @@
+; Check lgc.rt.trace.ray dialect is being generated.
+
+; BEGIN_SHADERTEST
+; RUN: amdllpc --gpurt-library=%S/Shader_GpurtLibrary.spv --print-after=llpc-spirv-lower-translator -gfxip 10.3 -o /dev/null 2>&1 %s | FileCheck -check-prefixes=SHADERTEST %s
```

Comment:
Why is the `--gpurt-library` needed here? It should be possible to run raytracing `.pipe` files now, using the GPURT library from the gpurt_shim that's linked into the `amdllpc` binary. That would be preferable over adding the (fairly large) SPIR-V binary to the repository.

Reply:
Indeed. Previous logic for LLPC_CLIENT_INTERFACE_MAJOR_VERSION < 62 always assume GPURT library is given in `.pipe`, I change that in latest commit.